### ISSUE TITLE
Package gone from AUR

### ISSUE THREAD

**Issue by robertrossmann** [davispuh/open-vm-tools-dkms#13]
This package is no longer available on AUR - is this intentional? Without it, there seems to be no way of making shared folders work on ArchLinux.

I currently maintain a relatively popular Vagrant box with support for VirtualBox, Parallels and VMWare, but I cannot make new releases without shared folder support. :(

If you could publish this package to AUR again or help me find another alternative for shared folders support I would be very grateful.

Thanks!

**Comment by davispuh** [comment]
When AUR was migrated from AUR3 to AUR4 then all old packages were removed and I didn't add it.

Anyway now I fixed it for linux 4.1 and pushed to AUR, but I don't use it anymore and I probably won't update it if it breaks because I don't have time for that, especially as I don't use it.

I recommend using CIFS (Samba) or NFS because VMware doesn't support newer kernels and we've to manually update patches whenever something is changed in newer kernel.
